subsys/sd/mmc.c

```diff
@@ -14,6 +14,22 @@
 #include "sd_ops.h"
 #include "sd_utils.h"
 
+#define MMC_EXT_CSD_BYTES 512
+#define MMC_SWITCH_8_BIT_BUS_ARG (0xFC000000 & (0U << 26)) + \
```

Comment:
May want to add a comment as to why these aren't in the spec file (assuming it is because they are specific command argument values)